### PR TITLE
Add placeholder for Amber login button and NIP-55 functionality

### PR DESCRIPTION
**Pull request by aybanda**
# Add Amber Login Button and NIP-55 Placeholder

## Changes
- Added "Log in with Amber" button to the login screen
- Implemented placeholder for NIP-55 functionality
- Created `Nip55Service` with stub methods for handling NIP-55 intents
- Added deep link handler for potential NIP-55 URLs

## Implementation Details
- New button aligns with existing login options
- `handleAmberLogin` function logs action and calls placeholder `Nip55Service` method
- Deep link handler initialized in `App.svelte`

## Testing
- Verified button appearance and alignment on login screen

Screenshot 2024-09-20 at 11 10 50 PM

- Tested deep link handler initialization (full functionality pending)

## Notes
- This PR implements UI and structural changes only
- Full NIP-55 functionality requires implementation in welshman repository
- A separate PR for welshman will be created to add necessary interfaces and stub implementations

## TODO
- [ ] Implement full NIP-55 functionality in welshman
- [ ] Update Coracle to use welshman's NIP-55 implementation once available
- [ ] Comprehensive testing with actual NIP-55 compatible wallet

### CODE REVIEW

File: src/services/deepLinkHandler.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import {App} from "@capacitor/app"
+import {Nip55Service} from "./Nip55"
+
+export function initializeDeepLinkHandler() {
+  App.addListener("appUrlOpen", (data: {url: string}) => {
+    console.log("Deep link opened", data.url)
+
+    if (data.url.startsWith("nostr+")) {
+      Nip55Service.handleNip55Intent(data.url)
```
Comment (staab) [inline src/services/deepLinkHandler.ts:9]: Will this actually work? My understanding is that browsers aren't able to listen for intents. But maybe there's a shim in the library you're using?

Comment (aybanda) [reply src/services/deepLinkHandler.ts:9]: You're  right that standard web browsers can't directly listen for intents. However, we're leveraging Capacitor's capabilities here, which bridges the gap between web and native functionality.

In our Capacitor-based app, the `App.addListener('appUrlOpen', ...)` method actually taps into native deep linking mechanisms on both iOS and Android. This allows our app to respond to custom URL schemes like "nostr+" when it's running or launched via a deep link.
also this approach doesn't cover all deep linking scenarios, particularly when the app is completely closed on mobile devices. For those cases, we'll need to implement additional platform-specific code in our native projects.

If you are unsure of this library 
I could go back redo this with
https://github.com/IT-MikeS/capacitor-intents

Comment (aybanda) [reply src/services/deepLinkHandler.ts:9]: @staab 
also can you assign the issue to me I almost forget to mention it

Comment (staab) [reply src/services/deepLinkHandler.ts:9]: Looks like someone else beat you to the punch: https://github.com/coracle-social/coracle/issues/421#issuecomment-2367124851

I encourage you to contribute to their solution if you're interested in continuing work on this!

**Comment by staab** [comment]
Closed by https://github.com/coracle-social/coracle/pull/426

**Pull request closed without merging**